msg/versioned/AirspeedValidated.msg

```diff
@@ -1,22 +1,28 @@
+# Validated airspeed
+#
+# Provides information about airspeed (indicated, true, calibrated) and the source of the data.
+# Used by many services and published to MAVLink.
+
+
 uint32 MESSAGE_VERSION = 1
 
-uint64 timestamp				# time since system start (microseconds)
+uint64 timestamp  # [us] Time since system start
 
-float32 indicated_airspeed_m_s			# [m/s] Indicated airspeed (IAS), set to NAN if invalid
-float32 calibrated_airspeed_m_s     		# [m/s] Calibrated airspeed (CAS), set to NAN if invalid
-float32 true_airspeed_m_s			# [m/s] True airspeed (TAS), set to NAN if invalid
+float32 indicated_airspeed_m_s   # [m/s] [@invalid NaN] Indicated airspeed (IAS)
+float32 calibrated_airspeed_m_s  # [m/s] [@invalid NaN] Calibrated airspeed (CAS)
+float32 true_airspeed_m_s        # [m/s] [@invalid NaN] True airspeed (TAS)
 
-int8 airspeed_source				# Source of currently published airspeed values
-int8 DISABLED = -1
-int8 GROUND_MINUS_WIND = 0
-int8 SENSOR_1 = 1
-int8 SENSOR_2 = 2
-int8 SENSOR_3 = 3
-int8 SYNTHETIC = 4
+int8 airspeed_source        # Source of currently published airspeed values
+int8 DISABLED = -1          # Disabled
+int8 GROUND_MINUS_WIND = 0  # Ground speed minus wind
+int8 SENSOR_1 = 1           # Sensor 1
+int8 SENSOR_2 = 2           # Sensor 2
+int8 SENSOR_3 = 3           # Sensor 3
+int8 SYNTHETIC = 4          # Synthetic airspeed 
 
 # debug states
-float32 calibrated_ground_minus_wind_m_s 	# CAS calculated from groundspeed - windspeed, where windspeed is estimated based on a zero-sideslip assumption, set to NAN if invalid
-float32 calibraded_airspeed_synth_m_s		# synthetic airspeed in m/s, set to NAN if invalid
-float32 airspeed_derivative_filtered		# filtered indicated airspeed derivative [m/s/s]
-float32 throttle_filtered			# filtered fixed-wing throttle [-]
-float32 pitch_filtered				# filtered pitch [rad]
+float32 calibrated_ground_minus_wind_m_s  # [m/s] [@invalid NaN] CAS calculated from groundspeed - windspeed, where windspeed is estimated based on a zero-sideslip assumption
+float32 calibraded_airspeed_synth_m_s     # [m/s] [@invalid NaN] Synthetic airspeed
```

Comment:
Note typo in field name "calibraded". Don't know if worth fixing in this round.

Reply:
No we will wait with fixing it until we change the message anyway, as otherwise we'd need to add a message translation for it extra.

Reply:
But thanks for pointing it out!